src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1244,6 +1249,28 @@ theorem lsub_eq_of_range_eq {ι ι'} {f : ι → ordinal} {g : ι' → ordinal}
 theorem lsub_nmem_range {ι} (f : ι → ordinal) : lsub f ∉ set.range f :=
 λ ⟨i, h⟩, h.not_lt (lt_lsub f i)
 
+theorem lsub_typein (o : ordinal) : lsub.{u u} (typein o.out.r) = o :=
```

Comment:
Should such results be marked simp lemmas? It looks like it should to me. But I don't know this part of the library, and whether there is a reason not to mark such things as simp lemma.

Reply:
I believe that, on the PR where these come from, they've been made into simp lemmas already.

Reply:
Apparently not, I've fixed this.